zone/loottables.cpp

```diff
@@ -54,6 +54,13 @@ void ZoneDatabase::AddLootTableToNPC(NPC* npc,uint32 loottable_id, ItemList* ite
 	if (!lts)
 		return;
 
+
+	if(!content_service.DoEnabledFlagsPass(lts->content_flags))
```

Comment:
Per my comments earlier around having one filtering function, we should have a function here that passes in the 4 filtering columns from the table

```
min_expansion
max_expansion
content_flags
content_flags_disabled
```

Pass all 4 to a superset filtering function that can determine whether or not we should return

eg


```cpp
!content_service.DoesPassContentFiltering(min_expansion, max_expansion, content_flags, content_flags_disabled)
```

Reply:
Yes, I will update to do this